.github/workflows/sync-docs-from-node.yml

```
@@ -26,878 +26,342 @@ on:
+    needs: [prepare, aggregate-results, generate-docs]
+    if: always() && needs.prepare.outputs.should_continue == 'true' && (needs.aggregate-results.result == 'success' || needs.generate-docs.result == 'success')
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout documentation repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)